### ISSUE TITLE
MySQL ingestion fails with the Airflow schema.

### ISSUE THREAD

**Issue by amiorin** [open-metadata/OpenMetadata#1391]
I'm running openmetadata-ingestion to ingest the Airflow and Openmetadata schema in MySQL and I get this error.

![image](https://user-images.githubusercontent.com/32617/143429544-31fb0b3e-6660-421f-9934-5820b3cb0c39.png)

My guess is that we need to add support for the type BLOB
```sql
-- auto-generated definition
create table dag_pickle
(
    id           int auto_increment
        primary key,
    pickle       blob         null,
    created_dttm timestamp(6) null,
    pickle_hash  bigint       null
);
```

**Comment by harshach** [comment]
@ayush-shah lets write that unit test as we talked about. collect all the types and add a unit test for creating a table. We should get validation error if the mapping doesn't work

**Comment by ayush-shah** [comment]
Should be fixed with #1405.
Added support for the datatypes.
